include/kframework/foundry.md

```diff
@@ -1168,9 +1192,6 @@ If the production is matched when no prank is active, it will be ignored.
 - selectors for unimplemented cheat code functions.
 
 ```k
-    rule selector ( "expectRegularCall(address,bytes)" )        => 3178868520
-    rule selector ( "expectNoCall()" )                          => 3861374088
-    rule selector ( "symbolicStorage(address)" )                => 769677742
```

Comment:
Is there a reason for this remove?

Reply:
That was the result of running `make tests/foundry/foundry.k.check CHECK=cp`.